src/components/CourseInfo/CourseInfo.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useContext } from 'react';
+
+import './CourseInfo.scss';
+import { getAuthors } from '../../helpers/courses.js';
+import { useParams } from 'react-router';
+import { mockedAuthorsList } from '../../constants';
+import CourseCard from '../Courses/components/CourseCard/CourseCard';
+import { Context } from '../../Store';
+
+const CourseInfo = () => {
+	const params = useParams();
```

Comment:
this is good, but it will be even better if you destructure whatever you need from params i.e. courseId